TerminalDocs/troubleshooting.md

```diff
@@ -70,3 +70,33 @@ On the other hand, if you do use this hotkey feature for multiple input language
 ## The text is blurry
 
 Some display drivers and hardware combinations do not handle scroll and/or dirty regions without blurring the data from the previous frame. To mitigate this problem, you can add a combination of [these global rendering settings](./customize-settings/global-settings.md#rendering-settings) to reduce the strain placed on your hardware caused by the terminal text renderer.
+
+## My colors look strange! There are black bars on my screen!
+
+> [!NOTE]
+> This applies only to version 1.2+ of Windows Terminal. If you are seeing color issues in Windows Terminal 1.0 or 1.1, or issues that are not captured here, please file a bug.
+
+Windows Terminal 1.2 and beyond have an improved understanding of certain application color settings. Because of this improved understanding, we have been able to remove a number of compatibility blocks that resulted in a poorer user experience. Unfortunately, there is a small number of applications that may experience issues.
```

Comment:
```suggestion
Windows Terminal 1.2 and beyond has an improved understanding of certain application color settings. Because of this improved understanding, we have been able to remove a number of compatibility blocks that resulted in a poor user experience. Unfortunately, there are a small number of applications that may experience issues.
```

Reply:
I'm not sure on all of these. first has, yes, last `are` is iffy because it refers to a [number] rather than an [applications]

Reply:
Oh now I'm unsure 😂

Reply:
how about this one w/ the first two from your suggestion?

```suggestion
Windows Terminal 1.2 and beyond has an improved understanding of certain application color settings. Because of this improved understanding, we have been able to remove a number of compatibility blocks that resulted in a poor user experience. Unfortunately, there is a small number of applications that may experience issues.
```

Reply:
Okay you're right. Wow that messed with my brain.